packages/cli/test/cases/develop.default/import-map.snapshot.json

```diff
@@ -25,8 +83,66 @@
   "@shoelace-style/shoelace/dist/utilities/localize.d.ts": "/~/Users/owenbuckley/Workspace/project-evergreen/greenwood/node_modules/@shoelace-style/shoelace/dist/utilities/localize.d.ts",
   "@shoelace-style/shoelace/dist/utilities/localize.js": "/~/Users/owenbuckley/Workspace/project-evergreen/greenwood/node_modules/@shoelace-style/shoelace/dist/utilities/localize.js",
   "@shoelace-style/shoelace/dist/react": "/~/Users/owenbuckley/Workspace/project-evergreen/greenwood/node_modules/@shoelace-style/shoelace/dist/react/index.js",
+  "@shoelace-style/shoelace/dist/react/alert": "/~/Users/owenbuckley/Workspace/project-evergreen/greenwood/node_modules/@shoelace-style/shoelace/dist/react/alert",
```

Comment:
Ok, so I think I see the issue here.

Given this pattern of [`/dist/react/*`](https://app.unpkg.com/@shoelace-style/shoelace@2.20.1/files/package.json#L28), `fs.glob` will match, but to only _one_ depth.  

So while the full path to reach is _dist/react/alert/index.js_, the single `*` will effectively stop just short of reaching to the final depth and stop at _dist/react/alert/_
https://app.unpkg.com/@shoelace-style/shoelace@2.20.1/files/dist/react/alert
```sh
@@@@@ {
  pathname: '/Users/owenbuckley/Workspace/project-evergreen/greenwood/node_modules/@shoelace-style/shoelace/',
  key: './dist/react/*'
}
{ match: 'dist/react/alert', relativePath: 'dist/react/alert' }
```